cmp/grid/GridModel.js

```diff
@@ -1544,7 +1581,7 @@ export class GridModel extends HoistModel {
 
 /**
  * @typedef {Object} GridAutosizeOptions
- * @property {GridAutosizeMode} [mode] - defaults to GridAutosizeMode.ON_SIZING_MODE_CHANGE.
+ * @property {GridAutosizeMode} [mode] - defaults to GridAutosizeMode.MANAGED.
```

Comment:
Like the new default, but a little bit worried that on introduction we are going to have existing grids that have been customized get aggressively autosized.  Maybe this is a one-time hit that we are willing to take? Not sure